samples/bluetooth/cap_acceptor/src/main.c

```diff
@@ -228,11 +227,21 @@ static int register_pac(enum bt_audio_dir dir, enum bt_audio_context context,
 		return err;
 	}
 
-	err = bt_pacs_set_location(dir, BT_AUDIO_LOCATION_MONO_AUDIO);
-	if (err != 0) {
-		LOG_ERR("Failed to set location: %d", err);
+	if (dir == BT_AUDIO_DIR_SOURCE) {
+		err = bt_pacs_set_location(dir, BT_AUDIO_LOCATION_FRONT_CENTER);
+		if (err != 0) {
+			LOG_ERR("Failed to set source location: %d", err);
 
-		return err;
+			return err;
+		}
+	} else {
+		err = bt_pacs_set_location(dir, BT_AUDIO_LOCATION_FRONT_LEFT |
+							BT_AUDIO_LOCATION_FRONT_RIGHT);
+		if (err != 0) {
+			LOG_ERR("Failed to set sink location: %d", err);
+
+			return err;
+		}
```

Comment:
I would prefer if `location` was added as a parameter to the function instead, so it's clearer from where it is being called, what location is being set for which direction, rather than this implicit choice